### PR TITLE
BaSP-TG-44: Shared Componentes Success/Error Modal

### PR DESCRIPTION
**Pull request by AlejoGF**
• Modal Success/Error tested only on Employee
• Modal Success/Error regarding the scroll, that depends on Javier's modal form

### CODE REVIEW

**Review by agusting19 — COMMENTED** [review]
Only one thing to correct. The rest seems to be nice

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import { useEffect, useState } from 'react';
 import styles from './employees.module.css';
-import List from './List/List';
-import FormModal from './FormModal';
 import Preloader from '../Shared/Preloader/Preloader';
+import Table from '../Shared/Table/Table';
```
[the remark this answers is not in the document]
Comment (Javier-Cervera) [reply src/Components/Employees/index.js:4]: I understand from how it's implemented that it needs a working modal, and that modal needs a list/table.

Maybe we could merge it after the table and modal form.

[the remark this answers is not in the document]
Comment (agusting19) [reply src/Components/Employees/index.js:4]: It is well implemented, it just does not seem like good practice to me.
But you're right, we can implement it after merging the table.

**Review by Javier-Cervera — APPROVED** [review]
Ready for QA!

**Review by lucianofrailelopez — APPROVED** [review]

**Comment by LeonelListro** [comment]
# Test Results

Tested locally on https://alexis-trackgenix-app-git-feature-tg-44-modal-2c20f2-basp-m2022.vercel.app/employees

**Test Results:** Failed ⚠️

### Bugs found:

1. Modal Succes should be smaller and title and message should be center.

Sin título
Sin título

2. The margin between the message and the buttons should be bigger and tittle should be centered.

image

3. (Suggestion) Add cross button to the top right of the modal.

172952674-5899e57a-48b6-4ab9-bcfb-6b019196e209

**Review by LeonelListro — CHANGES_REQUESTED** [review]
Refer to https://github.com/BaSP-m2022/alexis-trackgenix-app/pull/17#issuecomment-1151670576

**Review by LeonelListro — APPROVED** [review]

**Pull request merged**